src/main/java/cz/sparko/boxitory/service/BoxRepository.java

```diff
@@ -37,7 +38,7 @@
      * @param boxProvider provider of box to find
      * @param boxVersion  version of box to find
      * @return {@link BoxStream} found by given parameters, {@link Optional#empty} when box file not found
-     * @throws cz.sparko.boxitory.conf.NotFoundException when box does not exist or don't have given provider or dont
+     * @throws NotFoundException when box does not exist or don't have given provider or dont
```

Comment:
please, make this `@link`, same as in `latestVersionOfBox` javadoc

Reply:
Done